reconcile/utils/terrascript_aws_client.py

```diff
@@ -4649,6 +4648,10 @@ def populate_tf_resource_alb(self, spec, ocm_map=None):
         if idle_timeout:
             values["idle_timeout"] = idle_timeout
 
+        ip_address_type = resource.get("ip_address_type")
+        if ip_address_type:
+            values["ip_address_type"] = ip_address_type
```

Comment:
If unset I'm assuming this will just be ipv4?

Reply:
As discussed I've set the default to "ipv4" in code to match the apparent default from the AWS console

Neither the aws docs or the terraform provider docs mention a default